examples/production/run-tests.sh

```diff
@@ -2,6 +2,9 @@
 
 set -xe
 
+UPGRADE_VERSION=v0.13.0
```

Comment:
Is this intended to be the latest version of the chart?

Reply:
ultimately I think? Was thinking we'd bump it on release for now, and work on an auto detection script that would set it to whatever is latest automatically in a future pr?

Reply:
haha read my mind with the auto detection.